manifests/init.pp

```diff
@@ -239,15 +238,15 @@
   }
   else {
     exec { 'Auto-generate JWK Key store file':
-      command  => "keytool -genkeypair -alias JWK -keyalg RSA -noprompt -dname 'CN=${server_base_url}, OU=ID, O=IRIDA, L=IRIDA,S=IRIDA,\
-       C=CA' -keystore ${jwk_key_store_path} -validity 3650 -storepass ${jwk_key_store_password} -keypass ${jwk_key_store_password}\
+      command  => "export PASS= ${jwk_key_store_password} ; keytool -genkeypair -alias JWK -keyalg RSA -noprompt -dname \
```

Comment:
Instead of doing `export PASS=...` wouldn't it make more sense to set `environment` paramter?

i.e. 
`environment => ['PASS=...']`

https://puppet.com/docs/puppet/7/types/exec.html#exec-attribute-environment

Reply:
switch to environment attr.